lib/Stickersheet.js

```diff
@@ -0,0 +1,126 @@
+    
+    var trixels = {
+        t1: {},
+        t2: {},
+        t3: {},
+        t4: {},
+        t5: {},
+        t6: {}
+    }
+    
+    // Make it easier to grab values from the original trixel's viewBox
+    var viewBoxTmp = trixel.element.getAttribute("viewBox");
+    viewBoxTmp = viewBoxTmp.split(' ');
+    var trixelViewBox = {
+        x: viewBoxTmp[0],
+        y: viewBoxTmp[1],
+        width: viewBoxTmp[2],
+        height: viewBoxTmp[3]
+    }
+
+    var trixelWHratio = trixelViewBox.width / trixelViewBox.height;
+    
+    var trixelWidth = pageWidth * .3622; // Best guess based on mockup
+    var trixelHeight = trixelWidth / trixelWHratio;
+
+    function dupTrixel() {
+        // lodash's _.clone() method returns empty object if given a DOM node,
+        // even though these fake DOM nodes are of type Object so we do this.
+        var dupTrixel = document.createElement("svg");
+        dupTrixel.innerHTML = trixel.element.innerHTML
+        dupTrixel.setAttribute("viewBox", trixel.element.getAttribute("viewBox"));
+        dupTrixel.setAttribute("width", trixelWidth);
+        return dupTrixel;
+    }
+    
+    Object.keys(trixels).map(function(trixel) {
+        trixels[trixel] = dupTrixel()
+        svgContainer.appendChild(trixels[trixel]);
+    });
+
+    /**          The interesting stuff: positioning things          **/
+    
+    /*
+        Page layout:
+        
+        | t1 t2   |
+        |   t4 t3 |
+        | t5 t6   |
+        |         |
+          Trixel.io
+        
+        --
+        The layout is like this so I can set the X val of t4 based off of t3's X val.
+    */
+
+    var paddingBetweenTrixels = pageWidth * .03; // Best guess based on mockup
+    var paddingFromPageSides = pageWidth * .025; // Best guess based on mockup
+    var paddingFromPageTop = pageWidth * .03; // Best guess based on mockup
+    var rowOffset = pageHeight * .265;
+
+    var t1X = paddingFromPageSides
+    var t1Y = -pageHeight/2 + trixelHeight/2 + paddingFromPageTop;
+    trixels.t1.setAttribute("x", t1X);
+    trixels.t1.setAttribute("y", t1Y);
+
+    trixels.t2.setAttribute("x", t1X + trixelWidth + paddingBetweenTrixels);
+    trixels.t2.setAttribute("y", t1Y);
+
+    var t3X = pageWidth - trixelWidth - paddingFromPageSides;
+    var t3Y = t1Y + rowOffset; // Best guess based on mockup
+    trixels.t3.setAttribute("x", t3X);
+    trixels.t3.setAttribute("y", t3Y);
+
+    trixels.t4.setAttribute("x", t3X - trixelWidth - paddingBetweenTrixels);
+    trixels.t4.setAttribute("y", t3Y);
+
+    var t5X = t1X;
+    var t5Y = t3Y + rowOffset;
+
+    trixels.t5.setAttribute("x", t5X);
+    trixels.t5.setAttribute("y", t5Y);
+
+    trixels.t6.setAttribute("x", t5X + trixelWidth + paddingBetweenTrixels);
+    trixels.t6.setAttribute("y", t5Y);
+
+    /**                             **                              **/
+
+
+
+    var nl = "";
```

Comment:
Rather than copy/pasting from https://github.com/yarn-co/kepler/blob/master/lib/Compile.js#L222-L229, can you [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself) this up into a utility function that both `Compile` and `Stickersheet` can use?